frontend/src/app/modules/login/login/login.component.ts

```diff
@@ -47,7 +48,11 @@ export class LoginComponent implements OnInit {
     if (this.config.CAS_PUBLIC.CAS_AUTHENTIFICATION) {
       // if token not here here, redirection to CAS login page
       const url_redirection_cas = `${this.config.CAS_PUBLIC.CAS_URL_LOGIN}?service=${this.config.API_ENDPOINT}/gn_auth/login_cas`;
-      document.location.href = url_redirection_cas;
+      if (!this._authService.isAuthenticated()) {
+        // TODO: set the local storage item 'expires_at' in the API route "gn_auth/login_cas"
+        localStorage.setItem('expires_at', moment().add(1, 'days').toISOString());
```

Comment:
The 'expires_at' local storage item is being set in the frontend, but it's mentioned in the PR description that it should be set in the `gn_auth/login_cas` API route. This should be addressed to avoid potential confusion or bugs in the future. [important]